### PR TITLE
Disallow time limit + cumulative time limit. Fix #374.

### LINKED ISSUE #374: Clarify whether a regular and cumulative limits can exist together
Relevant Regulations:

> A1a) The organisation team must enforce time limits for attempts and/or rounds.
> A1a2) Cumulative time limits may be enforced (e.g. 3 results with a cumulative time limit of 20 minutes). The time elapsed in a DNF result counts towards the cumulative time limit.

So it is not clear whether cumilative limits exist instead of regular ones or together with them. In practice I didn't see organizers set them both (though one could say that if an organizer sets a cumulative limit and doesn't set a regular one, the regular limit is 10 minutes by default).

If it is allowed to use them together, as a side effect it is allowed to use cumulative limit for multi-bld.

Inspired by French championship 2016 report.

### PR DESCRIPTION
**Pull request by Claster**
Fixes #374.

### CODE REVIEW

File: wca-guidelines.md [hunk 1]
```diff
@@ -151,6 +151,8 @@ To be more informative, each Guideline is classified using one of the following
 - A1a2+) [ADDITION] In case of a cumulative time limit, the judge records the original recorded time for a DNF on the score sheet in parentheses, e.g. "DNF (1:02.27)".
 - A1a2++) [CLARIFICATION] Cumulative time limits may be enforced for a single round or a combination of rounds of different events.
 - A1a2+++) [CLARIFICATION] In case of a cumulative time limit, the result of each remaining attempt in the round is recorded as DNS (Did Not Start) after a competitor reaches the time limit.
+- A1a2++++) [ADDITION] If cumulative time limits for a single event or a combination of events are enforced, such events don't have regular time limits per solve.
+- A1a2+++++) [CLARIFICATION] Events where time limit per solve is defined by Regulations (Fewest Moves Solving and Multiple Blindfolded Solving) can't have cumulative time limits.
```
Comment (Laura-O) [inline wca-guidelines.md]: I think this should be either "a time limit" or "time limits".

Comment (AlbertoPdRF) [reply wca-guidelines.md]: Also, we should take into account #405 here and name the events correctly.

EDIT: Or just fix #405 after merging this one.

Comment (lgarron) [reply wca-guidelines.md]: Nit: "cannot" instead of "can't".

**Comment by Laura-O** [comment]
@Claster Can you please implement the suggestions so that we can merge this PR?

**Comment by AlbertoPdRF** [comment]
LGTM!

**Pull request merged**